tests_tf/test_attacks.py

```diff
@@ -454,119 +454,6 @@ def test_attack_strength_np_batched(self):
     new_labs = np.argmax(self.sess.run(self.model.get_logits(x_adv)), axis=1)
     self.assertLess(np.mean(feed_labs == new_labs), 0.1)
 
-
-class TestBasicIterativeMethod(CommonAttackProperties):
```

Comment:
I think instead of having a big section of diff here, this should just be a name change to "TestProjectedGradientDescent"

Reply:
Done